.viperlightignore

```diff
@@ -10,21 +10,34 @@ build
 lib
 
 # NOTICE files
-.*/NOTICE
+
```

Comment:
I think this file merge conflicts might have been resolved incorrectly. [This](https://github.com/aws-solutions/solution-spark-on-aws/blob/develop/.viperlightignore) is what the `develop` version of the file looks like. That should help resolve the viperlight failure that you saw.